client-device-connection-infinispan/src/main/java/org/eclipse/hono/deviceconnection/infinispan/client/InfinispanRemoteConfigurationOptions.java

```diff
@@ -177,4 +177,11 @@
      */
     @WithDefault("false")
     boolean useSsl();
+
+    /**
+     * Gets the value.
```

Comment:
IMHO this should be more specific to the purpose of the property

Reply:
I've adapted the javadoc - also of the other method containing the same sentence.